### DISCUSSION TITLE
"Give feedback" link is outdated

### DISCUSSION THREAD

**Discussion by armandgrillet** [community/community#18087] [Projects and Issues] [answered]
Hi, this link in boards redirects to https://github.com/github/feedback/discussions/categories/issues-feedback instead of redirecting to https://github.com/github-community/community/discussions/categories/issues :

<img width="160" alt="PS 2022-06-08 à 12 23 03" src="https://user-images.githubusercontent.com/2117580/172593985-70c77086-9078-4a28-95a4-5be60fb60b34.png">

**Comment by azenMatt** [comment, marked as answer]
Hi @armandgrillet - thanks for sharing this - we are rolling out a fix now 😄
